hayabusa/builtin/System/Sys_7045_Critical_LateralMovement-SMBEXEC.yml

```
@@ -1,12 +1,12 @@
 author: longmd - n0n$la$
 date: 2023/10/16
-modified: 2022/10/16
+modified: 2023/10/19
 
 title: SMBExec Lateral Movement
```

```suggestion
title: SMBExec Lateral Movement  (service installation)
```
With just this title, it's hard to know what exactly you are doing, so how about this way of writing?